src/ducks/apps/components/UninstallModal.jsx

```diff
@@ -56,9 +61,19 @@ export class UninstallModal extends Component {
               />
               {uninstallError && (
                 <p className="u-error">
-                  {t('app_modal.uninstall.message.error', {
-                    message: uninstallError.message
-                  })}
+                  {linkedAppError ? (
+                    <>
+                      {t('app_modal.uninstall.linked_app.error') + ' '}
```

Comment:
I find JSX ugly, with:
- no if/else
- the needs to put `(` and `)` for things on several lines inside `{ ... }`
- `{ ... }` must return a single node, and so I've used the `<> ... </>` hack to regroup the several nodes inside a single one
- I've tried to put an HTML entity to keep the space: `{t('app_modal.uninstall.linked_app.error')} &#32;`, but it was eaten by JSX
- and, in general, so many special characters (``<> {} () ` ` "" '' ?: &&``) in a few lines (when it breaks, it's hard to understand why)

Reply:
Was it wanted to keep this link inside the `<p>`? IMHO the good design here was to choose between display the error message in the `<p>` or display a markdown styled component to have the link in the text.
I'll push a change an we'll see 👍

Reply:
> Was it wanted to keep this link inside the `<p>`?

Yes, we want to keep the link inside the `<p>`.